substratevm/src/com.oracle.svm.core.posix/src/com/oracle/svm/core/posix/linux/LinuxSubstitutions.java

```diff
@@ -66,6 +72,46 @@ public static String mapLibraryName(String libname) {
     }
 }
 
+@Platforms(Platform.LINUX.class)
+@TargetClass(java.lang.Runtime.class)
+final class Target_java_lang_Runtime {
+
+    @Substitute
+    public int availableProcessors() {
+        if (SubstrateOptions.MultiThreaded.getValue()) {
+            LinuxSched.cpu_set_t cpuSet;
+            int size, res;
```

Comment:
Are you copying some C code?  Is there a reason not to declare `cpuSet`, `size`, and `res` where they are first needed?  (That would also declare them one per line. :-)